Python/ast_unparse.c

```diff
@@ -67,6 +95,7 @@ append_repr(_PyUnicodeWriter *writer, PyObject *obj)
     if (!repr) {
         return -1;
     }
+    repr = replace_inf_value(repr);
```

Comment:
Would this replace in the `"inf"` string literal?

`repr` can be NULL.

Reply:
Yes, i didn't notice that. I think i need to add a special case for tuples because it needs to replace in elements which are float or complex. (I am covering repr == NULL case) Thanks

Reply:
This bug has been fixed by checking obj type.